pitch.md

```diff
@@ -23,38 +79,101 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
 by describing which unicorn properties (horn color, tattoo, etc.) are most 
 important to users and what kinds of values those properties might have._ 
 
+For now, and possibly always, I want to focus this language on beginners.
+With that in mind, I think the user will only be able to describe a few higher
+level LED actions like cascade, wave, fade, flash, and strip color. Various
+commands will require different properties but some would be color of the
+action (or range of colors for fade etc.), for how long the action should last,
+the brightness of the LEDs, and how wide the action should be - a 6 led wide 
+cascade or a single led etc.
+
+The value of the properties vary but time will be a length in seconds or 
+minutes or both. Color will either be a single RGB/HSV/HSL color or a range of 
+them and width will be an integer. 
+
+Additionally, there will be a way connect one composition to another. Similar
+to a `Jump` in Assembly, this will be the main way of connecting smaller
+compositions together into a more complex one. 
```

Comment:
Composing elements and connecting the compositions is an important part of a DSL (or perhaps any language) in my opinion.